### PR TITLE
Filter out private repos

### LINKED ISSUE #65: Don't include private repos in report.json
Search for `"isPrivate": true` and find 19 private repos listed. It doesn't look like any sensitive information is included, but it's probably best to not expose the existence of these repos at all.

### LINKED ISSUE #67: Label extraction is unreliable
From one run to the next, the list of labels extracted varies without good reason (e.g. https://github.com/w3c/validate-repos/commit/04043af092a5df5836ecae15f43b57fe697296b4#diff-76a84217c1c1e1d7ecf1a6de26061bb3 ).

### PR DESCRIPTION
**Pull request by dontcallmedom**
close #65

### CODE REVIEW

**Comment by vivienlacourba** [comment]
@dontcallmedom I believe this is for issue #65 and not #67 (please update PR description accordingly).

File: validate.js [hunk 1]
```diff
@@ -235,7 +235,7 @@ async function validate() {
   const results = {};
   results.errors = allerrors;
   results.timestamp = new Date();
-  results.repos = allrepos;
+  results.repos = allrepos.filter(r => !r.isPrivate);
```
Comment (foolip) [inline validate.js]: To save some work and not have the condition twice, can you instead filter in `async function sequenced`?

Comment (foolip) [reply validate.js]: @dontcallmedom did you mean to resolve this?

Comment (dontcallmedom) [reply validate.js]: doh, my commit had failed! fixed now

**Pull request merged**